bitgo-java-api/src/test/java/com/bitso/bitgo/util/TestClientV2.java

```diff
@@ -134,7 +135,18 @@ public void listWalletTransfers() throws IOException {
 //            resp.getTransfers().remove(0);
 //        }
         System.out.println("resp json = " + SerializationUtil.mapper.writeValueAsString(resp));
+    }
 
+    @Test
+    public void getWalletTransfers() throws IOException {
+//        try {
```

Comment:
Maybe lose this if no more tests are being done with the getWalletTransferId

Reply:
done